ggml-common.h

```diff
@@ -17,7 +17,10 @@
 #elif defined(GGML_COMMON_IMPL_CUDA)
 #include <cstdint>
 
-#define GGML_TABLE_BEGIN(type, name, size) static const __device__ __constant__ type name[size] = {
+// So, I'm not sure if there are GPU's out there that like having the i-quant data in
+// constant memory. Mine (RTX-4080) definitely does not like it.
+//#define GGML_TABLE_BEGIN(type, name, size) static const __device__ __constant__ type name[size] = {
+#define GGML_TABLE_BEGIN(type, name, size) static const __device__ type name[size] = {
```

Comment:
I just did tests on RTX 3090, RTX 4090 and A100 and on all of them it is significantly faster to not have the `__constant__` specifier, so it's not just RTX 4080 related